app/models/study_downloader.rb

```diff
@@ -0,0 +1,11 @@
+class StudyDownloader
+    def self.download(nct_ids)
+      
+      nct_ids.each do |nct_id|
+        record = StudyJsonRecord.find_by(nct_id: nct_id) || StudyJsonRecord.create(nct_id: nct_id, content: {})
+        record.update_from_api
```

Comment:
this will download the study using the old api, we want to have some code here that downloads the study using the new api, we want to use the class `ClinicalTrialsApiV2` to download the json and we want to store it in StudyJsonRecord with version '2'.

`record.update_from_api` will call our current method which is using version 1 of the api.

Reply:
> this will download the study using the old api, we want to have some code here that downloads the study using the new api, we want to use the class `ClinicalTrialsApiV2` to download the json and we want to store it in StudyJsonRecord with version '2'.
> 
> `record.update_from_api` will call our current method which is using version 1 of the api.

Thank you for review. Everything was fixed, please review it again.